### ISSUE TITLE
NVDA announces Checked/Un Checked instead of Selected/ Un Selected for choiceGroup

### ISSUE THREAD

**Issue by shyama1208** [OfficeDev/office-ui-fabric-react#5716]
### Bug Report 
 
- __Browser and OS versions__: Firefox
Screen Reader:NVDA
 
#### Describe the issue:
NVDA announces checked/unchecked for default choice group items instead of selected/un selected.
https://developer.microsoft.com/en-us/fabric#/components/choicegroup
 
#### Actual behavior:
 
NVDA should announce checked/un checked
 
#### Expected behavior:
NVDA should announce selected/un selected . 
Note: Narrator announces selected/un selected

**Comment by cliffkoh** [comment]
Hi @shyama1208, our primary support scenario is Narrator on Edge. We're happy to make fixes in cases where we are not following Aria spec, but if we are otherwise adhering to W3C Aria spec, we don't make fixes for specific screen readers (a bug should be filed with the screen reader instead). 

Please let us know if you have identified a specific code change that we should make to be in better adherence to W3C aria spec.

**Comment by philipzloh** [comment]
"NVDA should announce" -- this ultimately should not be the goal of Accessibility support in Fabric; we should be aiming for *correct markup*, and whatever user agents and ATs do is up to them.

`role="radio"` is expected to use the `aria-checked` state, which is implicitly provided to the user agent if we use the native `<radio>` element or `<input type="radio">`. No Fabric bug exists here, IMO.

**Comment by natalieethell** [comment]
As this is not a Fabric bug, I'll go ahead and close this issue.
